include/test/mir/test/event_matchers.h

```diff
@@ -35,6 +35,59 @@
 void PrintTo(MirEvent const& event, std::ostream *os);
```

Comment:
@graysonguarino could we please do the event matcher improvements as a separate PR?